recipes/locopy/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Looks like `LICENSE` is included in the 0.3.7 tarball, so you can remove from this PR.

Reply:
Done 👍